src/instructlab/cli/process/list.py

```diff
@@ -0,0 +1,23 @@
+# SPDX-License-Identifier: Apache-2.0
+
+# Third Party
+import click
+
+# First Party
+from instructlab import clickext
+from instructlab.process.process import list_processes
+from instructlab.utils import print_table
+
+
+@click.command(name="list")
+@clickext.display_params
+def list():
+    process_list = list_processes()
+    if process_list is not None and len(process_list) > 0:
+        print_table(["Type", "PID", "UUID", "Log File", "Runtime"], process_list)
+    else:
+        click.secho(
+            "No processes found in registry",
+            fg="red",
+        )
+        raise click.exceptions.Exit(1)
```

Comment:
nit: the existence of 0 detached processes isn't necessarily a failure

Reply:
would `Exit(0)` be good?

Reply:
It's a tricky situation - it _could_ indicate an issue, just not necessarily

Since there's no code failure in this logic path, I'd lean toward exiting with 0, but if this is being used for testing somehow feel free to leave it as 1 and we can revisit this later

Reply:
No its fine I can switch to 0 and just look for the proper log